packages/coil-extension/src/background/util/formatters.ts

```diff
@@ -0,0 +1,23 @@
+import { GraphQlClient } from '@coil/client'
+
+export async function formatTipSettings(
+  client: GraphQlClient,
+  token: string,
+  limitRemaining: any,
+  lastTippedAmount: any
+) {
+  // convert all tip settings from cents to dollars
+  // set default hotkey tip amounts since we don't yet get them from the user
+  // add feature flag and minTipLimit
+  const featureFlagResp = await client.featureEnabled(token, 'tipping-beta')
+  const minTipLimitResp = await client.minTipLimit(token)
```

Comment:
I know this is how I did it originally and I'm sorry :( Can we remove the api calls from the formatter and just add them to the `tipSettings` query, then only have the formatter util format the data?